### PR TITLE
#341 - Add Number Features to GRG Toolset and address labeling on Pro 2.2

### PR DESCRIPTION
**Pull request by csmoore**
#341 - Add Number Features to GRG Toolset and address labeling on Pro 2.2

## Description

#341 - Add Number Features to GRG Toolset and address labeling on Pro 2.2

## Related Issue

#341

## Motivation and Context

#341

## How Has This Been Tested?

Manually (Note: we were requested to not add any more unit tests to repo)

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: toolboxes/scripts/GRGTools.py [hunk 1]
```diff
@@ -498,6 +498,103 @@ def execute(self, parameters, messages):
...
+
+        output_features= arcpy.Parameter(name='output_features',
+                                         displayName='Output Numbered Features',
+                                         direction='Output',
+                                         datatype='DEFeatureClass',
+                                         parameterType='Optional',
+                                         enabled=True,
+                                         multiValue=False)
+
+        layerFile = "NumberedStructures.lyr"
```
Comment (csmoore) [inline toolboxes/scripts/GRGTools.py]: @BobBooth - here is the Pro vs. ArcMap layer file check [we discussed yesterday](https://github.com/Esri/military-tools-geoprocessing-toolbox/issues/341#issuecomment-396998765)

**Comment by csmoore** [comment]
@topowright - please see question/comment at: https://github.com/Esri/military-tools-geoprocessing-toolbox/issues/341#issuecomment-397589167

**Comment by csmoore** [comment]
While trying to repro @topowright 's possible problem I noticed another existing issue with the tool.

The python code assumes the "Features to Number" parameter is a Feature Layer (at the SelectByLayer step) - so if you supply a  feature class to the tool it will fail.

image

image

**Comment by topowright** [comment]
This pull request addresses the addition of the tool, but not the labels. I could be missing something, but wanted to add the statement for full clarification.

**Pull request merged**